clang/test/dpct/query_api_mapping/Thrust/thrust_api_test.cu

```diff
@@ -253,3 +253,210 @@
 // stable_partition-NEXT:  /*10*/ dpct::stable_partition(oneapi::dpl::execution::seq, data, data + N, S, is_even());
 // stable_partition-NEXT:  /*11*/ dpct::stable_partition(oneapi::dpl::execution::seq, host_data.begin(), host_data.begin() + N, host_S.begin(), is_even());
 // stable_partition-NEXT:  /*12*/ dpct::stable_partition(oneapi::dpl::execution::make_device_policy(q_ct1), device_data.begin(), device_data.begin() + N, device_s.begin(), is_even());
+
```

Comment:
As follow up, please also check the original CUDA code, and add   comment for some arguments.